### PR TITLE
OCPBUGS-13981: feat: update example to use new cpu partitioning mode

### PR DESCRIPTION
**Pull request by eggfoobar**

### CODE REVIEW

File: ztp/gitops-subscriptions/argocd/example/siteconfig/example-sno.yaml [hunk 1]
```diff
@@ -45,6 +45,7 @@ spec:
       - 1001:2::/112
     additionalNTPSources:
       - 1111:2222:3333:4444::2
+    cpuPartitioningMode: AllNodes # Initiates the cluster for workload partitioning.
```
Comment (pixelsoccupied) [inline ztp/gitops-subscriptions/argocd/example/siteconfig/example-sno.yaml:48]: Thanks for the example! Couple of Qs

- So to specify the cores we just simply use PerformanceProfile? Can we note that info here? 

- Can we update this doc too? https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/WorkloadPartitioning.md

**Review by imiller0 — APPROVED** [review]
/lgtm

**Pull request merged**